.github/workflows/ci.yaml

```diff
@@ -140,7 +140,12 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Public SQL Logic Tests
-        run: just slt -v 'sqllogictests/*'
+        run: |
+          if [[ "$ACTIONS_STEP_DEBUG" == "true" ]]; then
+            just slt -v 'sqllogictests/*'
+          else
+            just slt 'sqllogictests/*'
+          fi
```

Comment:
I spent a hot second trying to figure out if you could get the `slt` arg parser to check this (or check it in the `justfile` and we _could_ but it seems like a lot of worlk.

Reply:
I don't think we should couple the justfile to the CI & GH specific variables. 

IMO, this is a concern of the ci, not the script runner.

Reply:
@tychoish I updated the logic to use github actions conditionals instead of bash. LMK what you think.

Reply:
Looks good!